src/pages/community-partners.astro

```diff
@@ -19,7 +19,7 @@ const sponsors = await getCollection("sponsors", ({ data }) => {
 >
 
 
-  <Headline id="community-partners" title="Community partners"  center=true />
+  <Headline as="h1" id="community-partners" title="Community partners" />
   <p>
```

Comment:
this paragraph is misaligned:

![image](https://github.com/user-attachments/assets/68cf9ef7-c1fb-4e49-a8c2-52d92d536b4a)